<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sobre</class>
 <widget class="QDialog" name="sobre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btOk">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Informações Gerais</string>
   </property>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVersao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAno">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

